app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,84 +0,0 @@
-body {
```

Comment:
Nice! It's always fun to delete code 🗑 

If you wanted to take this a step further, you could delete `items.scss` and `pledges.scss` too (they're auto-generated and empty).

You could also prevent stylesheet stubs and the default scaffold stylesheet from being generated by tweaking the generator workflow:

```ruby
# config/application.rb

# ...
module Playtime
  class Application < Rails::Application
    # ...
    config.generators do |g|
      g.stylesheets     false
    end
    # ...
  end
end
```

Resource: http://guides.rubyonrails.org/generators.html#customizing-your-workflow